buildscripts/templates/SemanticAttributes.java.j2

```diff
@@ -39,6 +39,10 @@
 {%- set filtered_enums = filtered_attributes | selectattr('is_enum', 'equalto', true) | list %}
 {%- set excluded_namespaces_list = excluded_namespaces.replace("\"", "").split(' ') %}
 {%- if root_namespace not in excluded_namespaces_list and filtered_attributes | count > 0 %}
+{%- set name_map ={"messaging.client.id":"messaging_client_id2"} -%}
```

Comment:
manually defined and maintained maintained list of name overwrites. 

TODO:
- [ ] it should be easier and done with `-D`
- [ ]  it should be used in class names
- [ ] it should not affect stable artifact
- [ ] javadoc on incubating `ID2` should refer to stable `ID`